content/docs/rfcs/16/README.md

```diff
@@ -89,6 +89,18 @@ The `get_waku_v2_debug_v1_info` method retrieves information about a Waku v2 nod
 
 none
 
+### `get_waku_v2_debug_v1_version`
+
+The `get_waku_v2_debug_v1_version` method retrieves the version of a Waku v2 node as a string.
+The version SHOULD follow [semantic versioning](https://semver.org/).
```

Comment:
Would it make sense to specify the format of the response for the scenarios described here?

Reply:
done